### PR TITLE
MUMMNG-1681 : Dockerfile

### PR DESCRIPTION
**Pull request by timlevett**
Adds in a simple Dockerfile for future deployment options.

### CODE REVIEW

File: tomcat-users.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='utf-8'?>
+<tomcat-users>
+ <role rolename="manager-gui"/>
+ <role rolename="manager-gui"/>
```
Comment (apetro) [inline tomcat-users.xml]: Two `<role` elements with the same `rolename`?

File: tomcat-users.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+ <role rolename="manager-script"/>
+ <user username="admin" password="admin" roles="manager,manager-gui,manager-script" />
```
Comment (apetro) [inline tomcat-users.xml]: What blocks something other than the intended build script from using this Tomcat manager username and password to deploy something untoward to the running Docker instance?

Comment (timlevett) [inline tomcat-users.xml]: I was thinking of removing this, was just for dev

**Comment by apetro** [comment]
:+1:

**Pull request merged**